djot/inline.lua

```diff
@@ -318,7 +318,6 @@ Parser.matchers = {
         if openers and #openers > 0
             and openers[#openers][3] == "explicit_link" then
           local opener = openers[#openers]
-          local startdest, enddest = opener[5], pos
```

Comment:
just inlines the variable, to make the two paths look the same. 

I think the code is also a bit more clear without this extra aliasing.